scripts/deploy/govcms-backups-preserve

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
+GOVCMS_BACKUP_DIR=${GOVCMS_BACKUP_DIR:-/app/web/sites/default/files/private/backups}
+
+echo "GovCMS Deploy :: Preserve backups"
+
+if [ -f "$GOVCMS_BACKUP_DIR/pre-deploy-dump.sql.gz" ]; then
+  mv "$GOVCMS_BACKUP_DIR/pre-deploy-dump.sql.gz" "$GOVCMS_BACKUP_DIR/pre-deploy-dump-last-good.sql.gz"
```

Comment:
@steveworley Do we want to output something like
`Moved $GOVCMS_BACKUP_DIR/pre-deploy-dump.sql.gz to $GOVCMS_BACKUP_DIR/pre-deploy-dump-last-good.sql.gz`

also, do we want to add an output (`else`) if the dump does not exist. 

This way script will be more verbose which is what site owners probably want (so that scripts are not perceived as black boxes).

@tobybellwood @simesy 
What is your opinion on verbosity? (this script is just an example; the question addresses all deploy scripts)